assets/aws/Makefile

```diff
@@ -23,17 +23,14 @@ TELEPORT_UID ?= 1007
 INSTANCE_TYPE ?= t2.medium
 
 # Use comma-separated values without spaces for multiple regions
-# For now, limit AMI to regions with DynamoDB encryption at rest enabled
```

Comment:
was there a specific reason we limited the regions this way before?

Reply:
Back when DynamoDB encryption at rest was first launched, it was only supported in specific regions. It's now supported everywhere - see the bottom of https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/EncryptionAtRest.html